front/app/containers/Admin/projects/project/files/components/FileUploadWithDropzone.tsx

```diff
@@ -0,0 +1,182 @@
+import React, { useState } from 'react';
+
+import {
```

Comment:
Please ignore this "FileUploadWithDropzone" component.

After some new designs, it will be completely re-worked, but I wanted to do that in a separate branch so this one doesn't get too big to review :)